src/olympia/reviewers/tasks.py

```diff
@@ -22,3 +24,24 @@ def recalculate_post_review_weight(ids):
         for summary in summaries:
             summary.calculate_weight()
             summary.save()
+
+
+def get_recalc_needed_filters():
+    cv = '_current_version__'
+    summary_modified = F(f'{cv}autoapprovalsummary__modified')
+
+    return [
+        # Only recalculate add-ons that received recent abuse reports
+        # possibly through their authors.
```

Comment:
Can you give me a bit more context maybe? Past rejection history isn't taken into account specifically to issue recalculation - but that query only fetches add-ons that need their scores to be recalculated and doesn't update the risk.

Reply:
According to https://docs.google.com/spreadsheets/d/1CZEECHHqEmK87fNiWNslAsFQBL7zxYF-c63-zZVZuqw/edit#gid=491359892, the active user count as well as past rejection history are both used for calculating the risk.

Reply:
> According to docs.google.com/spreadsheets/d/1CZEECHHqEmK87fNiWNslAsFQBL7zxYF-c63-zZVZuqw/edit#gid=491359892, the active user count as well as past rejection history are both used for calculating the risk.

And they continue to be. #11790 asked specifically some pointers to issue a re-calculation of that risk score and deemed abuse reports and negative ratings worthy of that, that's simply what I implemented.

How would you take past rejection history and users into account? Using past rejection history to re-calculate the score doesn't really make sense to me, user-numbers maybe but we'd have to normalize that a bit and find a good middle ground as I'm sure it fluctuates. While ratings and abuse reports are steady and if they increase, are a good indicator for changing the risk score.

Reply:
You are right, rejecting history doesn't really apply, due to the way reviewers work. It will be taken into account during submission of a new version. Other than that, we'd only see a change if a reviewer disabled an intermediate but not the latest version (without looking at that latest version), which is quite unlikely.

For active users, I assume we store average values somewhere to generate the stats page? Could we compute `average_daily_users` for an older timeframe, compare it with the current one, and do a recalculation if the diff exceeds a certain threshold?

Reply:
I do think we have that data in `update_counts` but that makes the whole thing much more complicated.

Maybe do that in a separate step and discuss mentioned thresholds in a new issue so that we can tackle that separately?

Reply:
Sounds good. Can you mention that in the current issue and let Jorge know that we are going to do it in a follow-up?

Reply:
done